playbooks/ORCHESTRATION.md

```diff
@@ -127,9 +127,72 @@ Manually check for:
 
 <!-- TODO: Add playbook for disaster/recovery steps. -->
 
-### Hardware Key Signing
+## Step 3: Add delegation (Optional)
+
+This step will add a delegated role to the top-level targets that is
+controlled by an external GitHub repository. Coordinate with the
+delegation keyholder to run the `add-delegation` command (see
+https://github.com/sigstore/root-signing/blob/main/cmd/tuf/app/add-delegation.go).
+When creating the delegation with the command, a `target-meta`
+file has to be provided that lists the targets, similar to adding
+the top level targets, see
+[here](ORCHESTRATION.md#targets-and-delegation-configuration) for an
+example.
+After the delegation metadata is added and signed, the delegation
+keyholder should open a PR against the ceremony branch.
+The name of the PR MUST be `feat/add-delegation for
+<delegation-name>`.
+
+As part of running the `add-delegaton` command, a POP (proof of
+possession) has to be generated too. The computed POP should be stored
+in `${REPO}/staged/${FORK_POINT}.sig`, where the fork point is the
+fork point from `main` and the ceremony branch. This fork point is
+also used as the nonce when computing the POP (via `tuf key-pop-sign`,
+see below for an example).
+
+The delegation keyholder would run these commands (on a branch based
+on the ceremony branch):
+
+Create the delegation metadata
+```shell
+$ ./tuf add-delegation -name ${DELEGATION_NAME} \
+      -public-key ${PUB_KEY_REF} \
+      -target-meta delegate-meta.yaml \
+      -repository repository
+```
+
+```shell
+$ ./tuf sign \
+      -roles ${DELEGATION_NAME} \
+      -key ${KEY_REF} \
+      -repository ${REPO}
+```
+
+```shell
+$ FORK_POINT=$(git merge-base origin/main "${BRANCH}") \
+      ./tuf key-pop-sign \
+      -key ${KEY_REF} \
+      -challenge ${DELEGATION_NAME} \
+      -nonce ${FORK_POINT} > ${REPO}/staged/${FORK_POINT}.sig
+```
+Here `BRANCH` is the ceremony branch, not the branch for the delegation.
 
-Next, the root and targets file must be signed. Ask each root keyholder to follow [Signing root and targets](../KEYHOLDER.md#signing-root-and-targets).
+When the PR is created, it will trigger the POP verify
+[workflow](../.github/workflows/delegation-pop-verify.yml), and upon
+successfull verification it will post a comment to the PR with the
+verification output.
+
+To manually verify the POP, run `./scripts/dpop-verify.sh ${PR_NUM}
```

Comment:
Since most flows are using workflows so step-0.sh hasn't been mentioned, either mention at the pre-ceremony steps to run that, add

```suggestion
To manually verify the POP, run `./scripts/step-0.sh && ./scripts/dpop-verify.sh ${PR_NUM}
```